bricks/basic-bricks/src/quick-visit-menu/QuickVisitMenu.tsx

```diff
@@ -293,7 +293,7 @@ export function QuickVisitMenu(props: QuickVisitMenuProps): React.ReactElement {
           <MenuContainer
             allMenus={allMenus}
             searchPlaceholder={searchPlaceholder ?? t(K.ENTER_KEY_WORDS)}
-            favouriteMenus={favouriteMenus}
+            favouriteMenus={favouriteMenus ?? []}
```

Comment:
这里让 `MenuContainer` 必填 `favouriteMenus`
```suggestion
            favouriteMenus={favouriteMenus}
```